Libraries/Components/WebView/WebView.ios.js

```diff
@@ -109,7 +128,7 @@ var WebView = React.createClass({
     source: PropTypes.oneOfType([
       PropTypes.shape({
         /*
-         * The URI to load in the WebView. Can be a local or remote file.
+         * The URI to load in the web view. Can be a local or remote file.
```

Comment:
Could be either/or. Sometimes it's a more generic thing being discussed. In this case not.